routers/users.py

```
@@ -0,0 +1,369 @@
+            "nickname": new_user["nickname"],
+            "profileImage": new_user["profileImage"],
+            "createdAt": new_user["createdAt"]
+        }
+    }
+
+
+
+
+
+@router.post("/login", response_model=LoginResponse)
+async def login(form_data: OAuth2PasswordRequestForm = Depends()):
```

Ruff (0.14.13)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)